ea_checkspm.m

```diff
@@ -1,4 +1,14 @@
 function ea_checkspm
+% 
```

Comment:
Please provide a description of the function.

Reply:
Check if a proper SPM (version > 6906) is installed